### PR TITLE
Add documentation on the StatusService

### PR DESCRIPTION
**Pull request by iainx**
If https://github.com/mono/monodevelop/pull/3100 is merged, here is some documentation for it

### CODE REVIEW

File: developers/articles/status-service.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: Status Service
+redirect_from:
+  - /Developers/Articles/StatusService/
+  - /StatusService/
```
Comment (akoeplinger) [inline developers/articles/status-service.md:5]: those redirects are for keeping the links from the old MediaWiki website working, you don't really need to add them on a new page

File: developers/articles/status-service.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    icon = StatusService.ShowStatusIcon (image);
+    icon.Title = "Test Icon";
+    icon.Clicked += (o, e) => {
+        Console.WriteLine ("Test icon clicked");
+    };
+
+    ...
+
+    // To remove the icon when finished with
+    icon.Dispose ();
```
Comment (akoeplinger) [inline developers/articles/status-service.md:112]: missing backticks at the end to close the code block

**Review by akoeplinger — CHANGES_REQUESTED** [review]

**Comment by akoeplinger** [comment]
@iainx is this still in progress?

**Comment by iainx** [comment]
The code never got merged, and has dropped off the radar. I'll close this and reopen if the code ever gets merged

**Pull request closed without merging**